### PR TITLE
Update scripts.

### PR DESCRIPTION
**Pull request by stephenmichaelf**
Trying to get consistency in our CI scripts. I started with the tool lib and am copying that setup here.

Let me know if you have thoughts on this/want to change anything.

Planning to do this for all OSS libs.

### CODE REVIEW

File: azure-pipelines-steps.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+steps:
+
+- script: npm install
+  displayName: 'npm install'
+
+- script: npm run build
+  displayName: 'npm run build'
+
+# test with node 6
+- task: NodeTool@0
```
Comment (damccorm) [inline azure-pipelines-steps.yml]: I think if we do this we should matrix across different versions of node - probably 6 and 12, maybe 5 too (we currently do 5). Note that this need isn't consistent across all our OSS repos, e.g. in typed-rest-client we currently test as low as 4 I believe

Comment (damccorm) [reply azure-pipelines-steps.yml]: We might also want to matrix across more hosted platforms? E.g. windows-2019

**Comment by damccorm** [comment]
Also, this failed CI with a parse error

**Comment by stephenmichaelf** [comment]
Will reopen when I can look at this.

**Pull request closed without merging**